ada_url/ada_adapter.py

```diff
@@ -354,7 +386,8 @@ def parse_url(s, attributes=PARSE_ATTRIBUTES):
     return ret
 
 
-def replace_url(s, **kwargs):
+# FIXME constrain key of `**kwargs` by `URL_ATTRIBUTES`
```

Comment:
The docstring says that unknown keys are ignored; I'd like to preserve this behavior.

Reply:
I see, but typing isn't just about error checking; constraints make auto suggestions work better.

If you insist, I'll just delete the comment; or I'll use `Union[Literal["href"], ...]`.

Reply:
Let's remove the comments and create a GitHub issue to help make auto suggestions (what editor?) better.